src/PlaywrightSharp.ProtocolTypesGenerator/ProtocolTypesGeneratorBase.cs

```diff
@@ -43,5 +43,8 @@ protected void GenerateRequestDefinition(StringBuilder builder, string typeName)
 
         protected void GenerateResponseDefinition(StringBuilder builder, string typeName)
             => builder.AppendLine($"internal partial class {typeName}Response: I{Project}Response");
+
+        protected void GeneratePropertyDefinition(StringBuilder builder, string type, string name)
+            => builder.AppendLine($"[System.Text.Json.Serialization.JsonPropertyName(\"{name}\")] public {type} {name.ToPascalCase()} {{ get; set; }}");

```

Comment:
Shouldn't this be solved by the JSON serializer?

Reply:
for most cases yes. in firefox we have a `base64body` which is converted to `Base64Body` and deserialized as `base64Body` which is wrong. this way we won't have potential errors in the future due to this

Reply:
Boooo! 
Good solution 😉